README.md

```diff
@@ -101,7 +101,9 @@ These fields are merged with all library, executable, test, and benchmark compon
 | --- | --- | --- | --- |
 | `exposed` | · | | |
 | `exposed-modules` | · | All modules in `source-dirs` less `other-modules` less any modules mentioned in `when` | |
+| `generated-exposed-modules` | | `[]` | Added to `exposed-modules` and `autogen-modules`. Since `0.21.0`.
```

Comment:
- I would leave default empty instead of saying `[]`.  This is the convention used in the README so far.
- I renamed the last column from `Notes` to `Default inside conditional`, so your content does not fit that column anymore.  What I would do is add `exposed-modules and autogen-modules` to the second column (`Cabal`).  If we want to specify `since`, we may want to have a column for that as we have for `Top-level fields`.

Reply:
Regarding `since`, this will tentatively go into `0.22.0`.

Reply:
The columns only seem to have been changed in the 'Library' section. Is it deliberate that it's still 'Notes' for executables, tests and benchmarks, as well as the common fields? Also, I notice that only `other-modules` has (documented) behaviour; it possibly makes more sense to go back to 'Notes'?

Reply:
Yes, works for me.